doc/articles/create-an-app-vs2022.md

```diff
@@ -120,6 +120,17 @@ You're all set!
 > [!IMPORTANT]
 > Take a [look at our article](xref:Build.Solution.TargetFramework-override) in order to ensure that your solution is building and showing intellisense as fast as possible, and to avoid [this Visual Studio issue](https://developercommunity.visualstudio.com/t/Building-a-cross-targeted-project-with-m/651372?space=8&q=building-a-cross-targeted-project-with-many-target) (help the community by upvoting it!) where multi-targeted project libraries always build their full set of targets.
 
-## Explore
 
-Next, explore [Uno Platform features and architecture](xref:Uno.GetStarted.Explore).
\ No newline at end of file
+
+## Next Steps
+
+Now that you're Created and Debug the App.
+
+Learn more about:
```

Comment:
Maybe something like: "Now that you've created and debugged the App, learn more about:"

Reply:
Made in PR https://github.com/unoplatform/uno/pull/14774